http/src/main/resources/reference.conf

```diff
@@ -25,7 +25,9 @@ dataproc {
   # Unset to use project-level defaults. Note a VPC/subnet with the literal name "default" may not exist.
   #vpcNetwork = "default"
   #vpcSubnet = "default"
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dbb0929"
+  # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
+  legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dbb0929"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-test3" # TODO: update this with a valid image
```

Comment:
Just calling out this TODO - should be addressed before merge, right?

Reply:
@akarukappadath , can we use this image in production? Otherwise, I can try to build another usable image outside this PR.